pkg/containerd/opts/spec_unix.go

```diff
@@ -483,6 +484,14 @@ func WithResources(resources *runtime.LinuxContainerResources) oci.SpecOpts {
 		if limit != 0 {
 			s.Linux.Resources.Memory.Limit = &limit
 		}
+		if len(hugepages) != 0 {
```

Comment:
Why the extra check? `rage` handles `nil` and `len() == 0` and will simply not enumerate

Reply:
Agree.

Reply:
done